src/components/calendar/Mes.js

```diff
@@ -1,13 +1,16 @@
 import React, { Component } from 'react';
 import styled from 'styled-components'
-import Dia from './Dia'
 import {diasDaSemana} from '../../controller/calendarController';
 import './style.css';
 
 const DiaSemana = styled.span`
 width: 14.28571428571429%;
 `
-  
+const Dia = styled.span`
+    text-align: center;
+    width: 14.28571428571429%;
```

Comment:
que específico 👀 tem algum motivo?

Reply:
hahaha eu acabei deixando assim por conta da divsão de 100(tamanho ocupado pela div) pelos 7 dias, mas vi que se eu colocar 14.3% não tem nenhuma diferença visivel

Reply:
Boa! Quando for cálculos assim, podemos usar o display flex mesmo e deixar ele calcular pra gente o/